### ISSUE TITLE
Too low contrast for no access warning

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1258]
The link controlled by `DEVILRY_LACKING_PERMISSIONS_URL` that follows «You do not have access to anything in Devilry.» in the warning displayed for users without any roles in Devilry has too low contrast:

<img width="538" alt="Skjermbilde 2023-03-29 kl  10 35 51" src="https://user-images.githubusercontent.com/796053/228479200-f68d0296-d3c7-4b20-8469-4073a450642a.png">
